data_import/admin.py

```diff
@@ -62,5 +63,23 @@ def save_model(self, request, obj, form, change):
         super().save_model(request, obj, form, change)
 
 
+class AdminStandardizedFile(admin.ModelAdmin):
+    form = UploadForm
+
+    def save_model(self, request, obj, form, change):
+        upload = Upload.objects.create()
+        standardized_file = form.cleaned_data['standardized_file']
+        reporting_year = form.cleaned_data['reporting_year']
```

Comment:
These should both already be defined on the `obj` (I think), so you shouldn't need to reset them.